src/test/kotlin/no/nav/syfo/application/api/PodApiSpek.kt

```diff
@@ -0,0 +1,103 @@
+import io.ktor.http.*
+import io.ktor.routing.*
+import io.ktor.server.testing.*
+import no.nav.syfo.application.ApplicationState
+import no.nav.syfo.application.api.registerPodApi
+import no.nav.syfo.testhelper.TestDatabase
+import no.nav.syfo.testhelper.TestDatabaseNotResponding
+import org.amshove.kluent.shouldBeEqualTo
+import org.amshove.kluent.shouldNotBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+object PodApiSpek : Spek({
+
+    describe("Successful liveness and readiness checks") {
+        with(TestApplicationEngine()) {
+            start()
+            val database = TestDatabase()
+            application.routing {
+                registerPodApi(
+                    ApplicationState(
+                        alive = true,
+                        ready = true
+                    ),
+                    database,
```

Comment:
Endre gjerne til name parameters i denne testen(se isoppfolgingstilfelle for "nyeste" ktor-kode.)